language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/utils/CommonUtil.java

```diff
@@ -141,6 +141,10 @@
     public static final Path LS_STDLIB_CACHE_DIR = TEMP_DIR.resolve("ls_stdlib_cache").resolve(SDK_VERSION);
 
     public static final Path LS_CONNECTOR_CACHE_DIR = TEMP_DIR.resolve("ls_connector_cache").resolve(SDK_VERSION);
+    
+    public static final List<String> PRE_DECLARED_LANG_LIBS = Arrays.asList("lang.boolean", "lang.decimal",
+            "lang.error", "lang.float", "lang.future", "lang.int", "lang.map", "lang.object", "lang.stream",
+            "lang.string", "lang.table", "lang.typedesc", "lang.xml", "lang.annotations");
```

Comment:
Any specific reason to maintain a set of pre-declared lang libs?

Reply:
This is due to #26243 where the compiler now pre-declare langLibs with reserved keywords. So we do not need go add the auto import and also we do not need the suggestions at the import declaration.

Reply:
Thanks, eventually we need to incorporate this into auto-imports in code-actions.